jdk/make/lib/SoundLibraries.gmk

```diff
@@ -151,6 +151,10 @@ else
     LIBJSOUND_CFLAGS += -DX_ARCH=X_S390X
   endif
 
+  ifeq ($(OPENJDK_TARGET_CPU), aarch64)
+    LIBJSOUND_CFLAGS += -DX_ARCH=X_AARCH64
```

Comment:
Where is `X_AARCH64` defined?
Perhaps `X_AARCH64` (and `X_S390X`) should be added to `SoundDefs.h`?

Reply:
Something needs to be passed to `X_ARCH`, otherwise you see the error message "You need to define X_PLATFORM and X_ARCH outside of the source."
The `X_ARCH` value is not used at all, however.

Reply:
I see uses in `Platform.c`, e.g. https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/jdk/src/share/native/com/sun/media/sound/Platform.c#L47.

Reply:
SoundDefs.h has no line for IBM copyright.  Do I need to add the line?  What would be the first year in the line in that case?

Reply:
I added the definitions of `X_S390X` and `X_AARCH64` in SoundDefs.h.